src/platform/linux/kmsgrab.cpp

```diff
@@ -701,8 +708,8 @@ namespace platf {
             img_offset_x = crtc->x;
             img_offset_y = crtc->y;
 
-            this->env_width = ::platf::kms::env_width;
-            this->env_height = ::platf::kms::env_height;
+            this->env_width = ::platf::kms::env_monitors[monitor].width;
+            this->env_height = ::platf::kms::env_monitors[monitor].height;
```

Comment:
Is there any risk with this breaking if we disconnect or add a monitor after sunshine was started? Can some id be used instead of index to improve safety?

Reply:
Functions that hanlle monitor change should take action to keep env_monitors updated, otherwise it might cause other issues.